kalite/distributed/static/js/distributed/search/views.js

```diff
@@ -116,6 +116,14 @@ window.AutoCompleteView = BaseView.extend({
             return compvalue;
         });
 
+        // Then sort again by availability!
+        ids_filtered.sort(function(id1, id2) {
```

Comment:
Hrm - it seems that this sort destroys the order imposed by the previous sort?

Topic sandwiched between two exercises here:

![image](https://cloud.githubusercontent.com/assets/1680573/7973278/9aa89588-0a08-11e5-804c-e3a72056d93e.png)